### ISSUE TITLE
[REQUEST] Can you provide a preview image?

### ISSUE THREAD

**Issue by afmpjr** [lalizita/video-thumbs#1]
I'm not sure what to expect.

**Comment by lalizita** [comment]
Yes, the preview image will be served by the server command in the application
https://github.com/lalizita/video-thumbs/blob/main/cmd/server.go

**Comment by afmpjr** [comment]
I'm sorry was not clear.
My request is to have a screenshot of what output to expect by using this.
